building on windows

Is there a way to build kafkahq on windows, pretty much all of the assemble targets i run error out with

```
> Task :webpack FAILED
C:\src\kafkahq\node_modules\.bin\webpack:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at Module._compile (internal/modules/cjs/loader.js:895:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at internal/main/run_main_module.js:17:11
```

I know nothing about webpack, but it looks to be a platform tooling issue? I verified and both `dirname` and `sed` are available in the path using gnutools.

It seems that changing in `build.gradle`
`script = project.file("node_modules/.bin/webpack")`
to
`script = project.file("node_modules/webpack/bin/webpack.js")`

makes the build progress a bit further. But then i get presented with this:

```
08:25:17: Executing task 'assembleDist --stacktrace'...

....

ERROR in ./assets/css/vendor.scss
Module build failed (from ./node_modules/css-loader/dist/cjs.js):
ModuleParseError: Module parse failed: Unexpected character ' ' (1:0)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
    at handleParseError (C:\src\kafkahq\node_modules\webpack\lib\NormalModule.js:447:19)
    at C:\src\kafkahq\node_modules\webpack\lib\NormalModule.js:481:5
    at C:\src\kafkahq\node_modules\webpack\lib\NormalModule.js:342:12
    at C:\src\kafkahq\node_modules\loader-runner\lib\LoaderRunner.js:373:3
    at iterateNormalLoaders (C:\src\kafkahq\node_modules\loader-runner\lib\LoaderRunner.js:214:10)
    at C:\src\kafkahq\node_modules\loader-runner\lib\LoaderRunner.js:205:4
    at C:\src\kafkahq\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:73:15
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
 @ ./assets/css/vendor.scss

ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.ttf?v=4.7.0 1:0
Module parse failed: Unexpected character ' ' (1:0)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 9:41-122
 @ ./assets/css/vendor.scss

ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0 1:4
Module parse failed: Unexpected character ' ' (1:4)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 7:41-124
 @ ./assets/css/vendor.scss

ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.woff?v=4.7.0 1:4
Module parse failed: Unexpected character ' ' (1:4)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 8:41-123
 @ ./assets/css/vendor.scss

ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.eot?v=4.7.0 1:1
Module parse failed: Unexpected character '�' (1:1)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 5:41-122
 @ ./assets/css/vendor.scss

ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.eot 1:1
Module parse failed: Unexpected character '�' (1:1)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 6:41-114
 @ ./assets/css/vendor.scss
Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/dist/cjs.js??ref--6-2!node_modules/sass-loader/lib/loader.js??ref--6-3!assets/css/assets.scss:
     3 assets
    Entrypoint undefined = extract-text-webpack-plugin-output-filename
    [0] ./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/assets.scss 1.16 KiB {0} [built]
    [3] ./assets/img/icon.png 58 bytes {0} [built]
    [4] ./assets/img/icon.svg 58 bytes {0} [built]
    [5] ./assets/img/logo.svg 58 bytes {0} [built]
        + 2 hidden modules
Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/dist/cjs.js??ref--6-2!node_modules/sass-loader/lib/loader.js??ref--6-3!assets/css/main.scss:
    Entrypoint undefined = extract-text-webpack-plugin-output-filename
    [0] ./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/main.scss 109 KiB {0} [built]
        + 1 hidden module
Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/dist/cjs.js??ref--6-2!node_modules/sass-loader/lib/loader.js??ref--6-3!assets/css/vendor.scss:
     1 asset
    Entrypoint undefined = extract-text-webpack-plugin-output-filename
    [1] ./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss 1.01 MiB {0} [built]
        + 10 hidden modules
    
    ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.ttf?v=4.7.0 1:0
    Module parse failed: Unexpected character ' ' (1:0)
    You may need an appropriate loader to handle this file type.
    (Source code omitted for this binary file)
     @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 9:41-122
    
    ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0 1:4
    Module parse failed: Unexpected character ' ' (1:4)
    You may need an appropriate loader to handle this file type.
    (Source code omitted for this binary file)
     @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 7:41-124
    
    ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.woff?v=4.7.0 1:4
    Module parse failed: Unexpected character ' ' (1:4)
    You may need an appropriate loader to handle this file type.
    (Source code omitted for this binary file)
     @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 8:41-123
    
    ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.eot?v=4.7.0 1:1
    Module parse failed: Unexpected character '�' (1:1)
    You may need an appropriate loader to handle this file type.
    (Source code omitted for this binary file)
     @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 5:41-122
    
    ERROR in ./node_modules/font-awesome/fonts/fontawesome-webfont.eot 1:1
    Module parse failed: Unexpected character '�' (1:1)
    You may need an appropriate loader to handle this file type.
    (Source code omitted for this binary file)
     @ ./assets/css/vendor.scss (./node_modules/css-loader/dist/cjs.js??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./assets/css/vendor.scss) 6:41-114

> Task :webpack FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':webpack'.
> Process 'command 'node'' finished with non-zero exit value 2

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':webpack'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$3.accept(ExecuteActionsTaskExecuter.java:166)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$3.accept(ExecuteActionsTaskExecuter.java:163)
	at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:191)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:156)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:62)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:108)
	at org.gradle.api.internal.tasks.execution.ResolveBeforeExecutionOutputsTaskExecuter.execute(ResolveBeforeExecutionOutputsTaskExecuter.java:67)
	at org.gradle.api.internal.tasks.execution.ResolveAfterPreviousExecutionStateTaskExecuter.execute(ResolveAfterPreviousExecutionStateTaskExecuter.java:46)
	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:94)
	at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
	at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:95)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
	at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:43)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:355)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:343)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:336)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:322)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:134)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:129)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:202)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:193)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:129)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
Caused by: org.gradle.process.internal.ExecException: Process 'command 'node'' finished with non-zero exit value 2
	at org.gradle.process.internal.DefaultExecHandle$ExecResultImpl.assertNormalExitValue(DefaultExecHandle.java:409)
	at org.gradle.process.internal.DefaultExecAction.execute(DefaultExecAction.java:38)
	at org.gradle.process.internal.DefaultExecActionFactory.exec(DefaultExecActionFactory.java:145)
	at org.gradle.api.internal.project.DefaultProject.exec(DefaultProject.java:1117)
	at org.gradle.api.internal.project.DefaultProject.exec(DefaultProject.java:1112)
	at org.gradle.api.Project$exec$3.call(Unknown Source)
	at com.moowork.gradle.node.exec.ExecRunner.run(ExecRunner.groovy:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at com.moowork.gradle.node.exec.NodeExecRunner.doExecute(NodeExecRunner.groovy:36)
	at com.moowork.gradle.node.exec.ExecRunner.execute(ExecRunner.groovy:63)
	at com.moowork.gradle.node.exec.ExecRunner$execute.call(Unknown Source)
	at com.moowork.gradle.node.task.NodeTask.exec(NodeTask.groovy:100)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:103)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:49)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:42)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:28)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:717)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:684)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$5.run(ExecuteActionsTaskExecuter.java:476)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:461)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:444)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.access$200(ExecuteActionsTaskExecuter.java:93)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$TaskExecution.execute(ExecuteActionsTaskExecuter.java:237)
	at org.gradle.internal.execution.steps.ExecuteStep.lambda$execute$1(ExecuteStep.java:33)
	at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:33)
	at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:26)
	at org.gradle.internal.execution.steps.CleanupOutputsStep.execute(CleanupOutputsStep.java:58)
	at org.gradle.internal.execution.steps.CleanupOutputsStep.execute(CleanupOutputsStep.java:35)
	at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:48)
	at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:33)
	at org.gradle.internal.execution.steps.CancelExecutionStep.execute(CancelExecutionStep.java:39)
	at org.gradle.internal.execution.steps.TimeoutStep.executeWithoutTimeout(TimeoutStep.java:73)
	at org.gradle.internal.execution.steps.TimeoutStep.execute(TimeoutStep.java:54)
	at org.gradle.internal.execution.steps.CatchExceptionStep.execute(CatchExceptionStep.java:35)
	at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:51)
	at org.gradle.internal.execution.steps.SnapshotOutputsStep.execute(SnapshotOutputsStep.java:45)
	at org.gradle.internal.execution.steps.SnapshotOutputsStep.execute(SnapshotOutputsStep.java:31)
	at org.gradle.internal.execution.steps.CacheStep.executeWithoutCache(CacheStep.java:208)
	at org.gradle.internal.execution.steps.CacheStep.execute(CacheStep.java:70)
	at org.gradle.internal.execution.steps.CacheStep.execute(CacheStep.java:45)
	at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:49)
	at org.gradle.internal.execution.steps.StoreSnapshotsStep.execute(StoreSnapshotsStep.java:43)
	at org.gradle.internal.execution.steps.StoreSnapshotsStep.execute(StoreSnapshotsStep.java:32)
	at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:38)
	at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:24)
	at org.gradle.internal.execution.steps.SkipUpToDateStep.executeBecause(SkipUpToDateStep.java:96)
	at org.gradle.internal.execution.steps.SkipUpToDateStep.lambda$execute$0(SkipUpToDateStep.java:89)
	at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:54)
	at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:38)
	at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:76)
	at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:37)
	at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:36)
	at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:26)
	at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:90)
	at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:48)
	at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:69)
	at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:47)
	at org.gradle.internal.execution.impl.DefaultWorkExecutor.execute(DefaultWorkExecutor.java:33)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:140)
	... 34 more

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -132,7 +132,7 @@ module.exports = (env, argv) => {
                 {
                     test: /\.(jpe?g|png|gif|svg)$/,
                     exclude: [
-                        /\/fonts\/|\\fonts\\/
+                        /[\\/]fonts[\\/]/
                     ],
                     loader: 'file-loader',
                     options: {
@@ -143,7 +143,7 @@ module.exports = (env, argv) => {
                 {
                     test: /\.(eot|svg|ttf|woff|woff2)(\?v=\d+\.\d+\.\d+)?$/,
                     include: [
-                        /\/fonts\/|\\fonts\\/
+                        /[\\/]fonts[\\/]/
                     ],
                     loader: 'file-loader?name=font/[name].[ext]',
                 }

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -132,7 +132,7 @@ module.exports = (env, argv) => {
                 {
                     test: /\.(jpe?g|png|gif|svg)$/,
                     exclude: [
-                        /\/fonts\//
+                        /\/fonts\/|\\fonts\\/
                     ],
                     loader: 'file-loader',
                     options: {
@@ -143,7 +143,7 @@ module.exports = (env, argv) => {
                 {
                     test: /\.(eot|svg|ttf|woff|woff2)(\?v=\d+\.\d+\.\d+)?$/,
                     include: [
-                        /\/fonts\//
+                        /\/fonts\/|\\fonts\\/
                     ],
                     loader: 'file-loader?name=font/[name].[ext]',
                 }

